statsmodels/discrete/tests/test_sandwich_cov.py

```diff
@@ -451,6 +451,70 @@ def test_basic(self):
         assert_allclose(res1.params, res2.params, rtol=rtol)
         assert_allclose(res1.bse, res2.bse, rtol=1e-10)
 
+    def test_score_hessian(self):
+        res1 = self.res1
+        res2 = self.res2
+
+        # We need to fix scale in GLM and OLS,
+        # discrete MLE have it always fixed
+        if isinstance(res2.model, OLS):
+            kwds = {'scale': res2.scale}
+        else:
+            kwds = {}
+        if isinstance(res2.model, OLS):
+            sgn = + 1
+        else:
+            sgn = -1  # see #4714
+
+        score1 = res1.model.score(res1.params * 0.98, scale=res1.scale)
+        score2 = res2.model.score(res1.params * 0.98, **kwds)
+        assert_allclose(score1, score2, rtol=1e-13)
+
+        hess1 = res1.model.hessian(res1.params, scale=res1.scale)
+        hess2 = res2.model.hessian(res1.params, **kwds)
+        assert_allclose(hess1, hess2, rtol=1e-10)
+
+        if isinstance(res2.model, OLS):
+            # skip the rest
+            return
+        scoref1 = res1.model.score_factor(res1.params, scale=res1.scale)
+        scoref2 = res2.model.score_factor(res1.params, **kwds)
+        assert_allclose(scoref1, scoref2, rtol=1e-10)
+
+        hessf1 = res1.model.hessian_factor(res1.params, scale=res1.scale)
+        hessf2 = res2.model.hessian_factor(res1.params, **kwds)
+        assert_allclose(sgn * hessf1, hessf2, rtol=1e-10)
+
+    def test_score_test(self):
+        res1 = self.res1
+        res2 = self.res2
+
+        if isinstance(res2.model, OLS):
+            # skip
+            return
+
+        fitted = self.res1.fittedvalues
+        exog_extra = np.column_stack((fitted**2, fitted**3))
+        res_lm1 = res1.score_test(exog_extra, cov_type='nonrobust')
+        res_lm2 = res2.score_test(exog_extra, cov_type='nonrobust')
+        assert_allclose(np.hstack(res_lm1), np.hstack(res_lm2), rtol=5e-7)
+
+
+class TestGLMPoisson(CheckDiscreteGLM):
+
+    @classmethod
+    def setup_class(cls):
+        endog_count = np.random.poisson(endog)
+        cls.cov_type = 'HC0'
+
+        mod1 = GLM(endog_count, exog, family=families.Poisson())
+        cls.res1 = mod1.fit(cov_type='HC0')
+
+        mod1 = smd.Poisson(endog_count, exog)
+        cls.res2 = mod1.fit(cov_type='HC0')
+
+        cls.res1.rtol = 1e-11
```

Comment:
should this be `cls.rtol`?

Reply:
I don't remember why this is this way
but on line 450 it uses res1.rtol if available